concepts/05 Widgets/Map/10 Specify the Provider and Type.md

```diff
@@ -85,6 +139,74 @@ When using maps, you should include an API key that authenticates your applicati
         // ...
     })
 
+##### Vue
+
+    <template>
+        <DxMap
+            :zoom="10"
+            :center="centerCoordinates"
+            provider="bing"
+            :key="authentificationKeys"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxMap } from 'devextreme-vue/map';
+
+    export default {
+        components: {
+            DxMap
+        },
+        data() {
+            return {
+                centerCoordinates: { lat: 40.749825, lng: -73.987963 },
+                authentificationKeys: {
+                    bing: this.BING_API_KEY,
+                    google: this.GOOGLE_API_KEY,
+                    googleStatic: this.GOOGLE_STATIC_API_KEY
```

Comment:
These variables have to be declared somewhere.

Reply:
We decided to change the API keys placeholders to strings, as in the [demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Map/ProvidersAndTypes/React/Light/)